templates/examples/qml/main.cpp.tpl

```diff
@@ -56,6 +56,7 @@ int main(int argc, char *argv[]){
     // Create main app
     const QUrl url(QStringLiteral("qrc:/main.qml"));
     QGuiApplication app(argc, argv);
+    app.addLibraryPath(app.applicationDirPath() + "../../imports");

```

Comment:
This assumes that we always know the folder structure.
We can leave it but I'd like to have a comment that it is better to use the `QML2_IMPORT_PATH`